requirements/time_tracker_api/dev.txt

```diff
@@ -14,9 +14,6 @@ pytest-mock==2.0.0
 # Coverage
 coverage==4.5.1
 
-# Git hooks
```

Comment:
@jcalarcon98 why are you removing this lines?

Reply:
Explained in an internal conversation, solved 💯